test/e2e/seamless_migration_test.go

```diff
@@ -0,0 +1,234 @@
+				framework.WaitClusterRoleDisappearOnCluster(member1, clusterRoleName)
+			})
+		})
+
+		ginkgo.It("Verify migrate a ClusterRole from member cluster", func() {
+			ginkgo.By(fmt.Sprintf("Verify ClusterResourceBinding %s got Applied by overwriting conflict resource", bindingName), func() {
+				klog.Infof("Waiting to verify ResourceBinding %s got Applied by overwriting conflict resource", bindingName)
+				gomega.Eventually(func() bool {
+					framework.WaitClusterRolePresentOnClusterFitWith(member1, clusterRoleName, func(clusterRole *rbacv1.ClusterRole) bool {
+						return true
+					})
+					_, e1 := kubeClient.RbacV1().ClusterRoles().Get(context.TODO(), clusterRoleName, metav1.GetOptions{})
+					binding, e2 := karmadaClient.WorkV1alpha2().ClusterResourceBindings().Get(context.TODO(), bindingName, metav1.GetOptions{})
+
+					return e1 == nil && e2 == nil && len(binding.Status.AggregatedStatus) > 0 && binding.Status.AggregatedStatus[0].Applied
+				}, pollTimeout, pollInterval).Should(gomega.Equal(true))
+			})
+		})
+	})
+
+	ginkgo.Context("Test migrate namespaced resource: Service (NodePort)", func() {
+		var serviceName string
+		var service *corev1.Service
+		var pp *policyv1alpha1.PropagationPolicy
+		var bindingName string
+
+		ginkgo.BeforeEach(func() {
+			serviceName = serviceNamePrefix + rand.String(RandomStrLength)
+			service = helper.NewService(testNamespace, serviceName, corev1.ServiceTypeNodePort)
+			pp = helper.NewPropagationPolicy(testNamespace, service.Name, []policyv1alpha1.ResourceSelector{
+				{
+					APIVersion: service.APIVersion,
+					Kind:       service.Kind,
+					Name:       service.Name,
+				},
+			}, policyv1alpha1.Placement{
+				ClusterAffinity: &policyv1alpha1.ClusterAffinity{ClusterNames: []string{member1}},
+			})
+			pp.Spec.ConflictResolution = policyv1alpha1.ConflictOverwrite
+			bindingName = names.GenerateBindingName(service.Kind, service.Name)
+		})
+
+		ginkgo.BeforeEach(func() {
+			// Create Deployment in member1 cluster
+			framework.CreateService(member1Client, service)
+			// Create Deployment in karmada control plane
+			framework.CreateService(kubeClient, service)
+			// Create PropagationPolicy in karmada control plane without conflictResolution field
+			framework.CreatePropagationPolicy(karmadaClient, pp)
+
+			ginkgo.DeferCleanup(func() {
+				// Delete Service in karmada control plane
+				framework.RemoveService(kubeClient, testNamespace, serviceName)
+				// Delete PropagationPolicy in karmada control plane
+				framework.RemovePropagationPolicy(karmadaClient, testNamespace, pp.Name)
+
+				// Verify Service in member cluster will be deleted automatically after promotion since it be deleted from Karmada
+				klog.Infof("Waiting for Service deleted from cluster(%s)", member1)
+				framework.WaitServiceDisappearOnCluster(member1, testNamespace, serviceName)
+			})
+		})
+
+		ginkgo.It("Test migrate a Service from member cluster", func() {
```

Comment:
minor suggestion: to use a consistent tone and format such as "Verify xxx" or "Test xxx" instead of mixing "Verify xxx" and "Test migrate xxx".
```suggestion
		ginkgo.It("Verify migrate a Service from member cluster", func() {
```